src/DocumentFormat.OpenXml/OpenXmlReader.cs

```diff
@@ -241,6 +241,47 @@ public virtual bool HasAttributes
         /// <remarks>The new current element is the end of the element after LoadCurrentElement().</remarks>
         public abstract OpenXmlElement LoadCurrentElement();
 
+        /// <summary>
+        /// Loads the element at current cursor as the specified type.
+        /// </summary>
+        /// <typeparam name="T">The concrete type of the OpenXmlElement to load.</typeparam>
+        /// <returns>The OpenXmlElement that was loaded.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when the current is the end element or when the ElementType and <typeparamref name="T"/> do not match.</exception>
+        public virtual T LoadElement<T>()
```

Comment:
Is there a reason to make it virtual? Once we make something virtual, we cannot go back. It's OK to add virtual later, so unless you have a reason it should be virtual, this seems like something that doesn't need to be made virtual

Reply:
fair enough.